patches/chrome-common-media-cdm_host_file_path.cc.patch

```diff
@@ -1,16 +0,0 @@
-diff --git a/chrome/common/media/cdm_host_file_path.cc b/chrome/common/media/cdm_host_file_path.cc
-index a454dd3450ae06949ab38f2d854b7a9f811604ff..193d1a682aa2e07ac479ed819033ceba9610a0ff 100644
---- a/chrome/common/media/cdm_host_file_path.cc
-+++ b/chrome/common/media/cdm_host_file_path.cc
-@@ -44,7 +44,11 @@ void AddCdmHostFilePaths(
- #if defined(OS_WIN)
- 
-   static const base::FilePath::CharType* const kUnversionedFiles[] = {
-+#if defined(BRAVE_CHROMIUM_BUILD)
```

Comment:
?

Reply:
This has been replaced with
<pre>
static const base::FilePath::CharType* const kUnversionedFiles[] = {
      chrome::kBrowserProcessExecutableName};
</pre>
which we override in `chromium_src/chrome/common/chrome_constants.cc`